Frontend/business-expense-manager/src/components/category-panel/category-panel.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="panel-wrapper">
+  <button (click)="close()" mat-flat-button class="close-button">
+    <mat-icon fontIcon="close"></mat-icon>
+  </button>
+  <div class="header-bar">
+    <h2>Category budget for: {{ categoryCard.title }} </h2>
+    <button (click)="editBudget()" aria-label="Edit category budget" class="add-button">
+      <mat-icon fontIcon="edit"></mat-icon>
+      <p>Edit budget</p>
+    </button>
+  </div>
+  <ng-container *ngIf="!error && dataSource?.data?.length; else noData">
+    <div class="mat-elevation-z8">
+      <table mat-table [dataSource]="dataSource">
+        <ng-container matColumnDef="Date Captured">
+          <th mat-header-cell *matHeaderCellDef>Date Captured</th>
+          <td mat-cell *matCellDef="let element">
+            {{ element.createdDatetime | date : "y-MM-d hh:mm" }}
+          </td>
+        </ng-container>
+
+        <ng-container matColumnDef="Category">
+          <th mat-header-cell *matHeaderCellDef>Category</th>
+          <td mat-cell *matCellDef="let element">{{ element.category }}</td>
+        </ng-container>
+
+        <ng-container matColumnDef="Goal">
+          <th mat-header-cell *matHeaderCellDef>Goal</th>
+          <td mat-cell *matCellDef="let element">{{ element.goal }}</td>
+        </ng-container>
+
+        <ng-container matColumnDef="Amount">
+          <th mat-header-cell *matHeaderCellDef>Amount</th>
+          <td mat-cell *matCellDef="let element">
+            {{ element.monetaryValue }}
+          </td>
+        </ng-container>
+
+        <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+        <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+      </table>
+
+      <mat-paginator [pageSizeOptions]="[10]" (page)="onPageChange($event)">
+        aria-label="Select page of periodic elements">
+      </mat-paginator>
+    </div>
+  </ng-container>
+</div>
+
+<ng-template #noData>
+  <p *ngIf="error">
+    An error has occurred fetching the cash flows. Please be patient while we
```

Comment:
I changed the error description to this: Would you mind changing it to this message:
  <p *ngIf="error">
    Cash flow history failed to load...
  </p>